src/content/overview.md

```diff
@@ -107,15 +107,15 @@ class Point {
 }
 ```
 
-{{site.alert.info}}
-  This example is running in an embedded [DartPad](/tools/dartpad).
-  You can also
-  <a href="{{site.dartpad}}/bc63d212c3252e44058ff76f34ef5730"
-  target="_blank" rel="noopener">open this example in its own window</a>.
-{{site.alert.end}}
+:::note
+This example is running in an embedded [DartPad](/tools/dartpad).
+You can also
+<a href="{{site.dartpad}}/bc63d212c3252e44058ff76f34ef5730"
+target="_blank" rel="noopener">open this example in its own window</a>.
+:::
 
 
-## Dart: The libraries {#libraries}
+## Dart: The libraries {:#libraries}
```

Comment:
I'm curious, is the `{:#...}` syntax required by 11ty?

Reply:
**Short explanation:** No, I configured it that way.

**Longer explanation:**

Compared to Jekyll, 11ty comes with a lot less in the box, but that also allows it to be more configurable.

The default Markdown package is `markdown-it` and this support comes from a plugin for that called [`markdown-it-attrs`](https://www.npmjs.com/package/markdown-it-attrs). It defaults to just `{` as the starting delimiter, but I [configured](https://github.com/dart-lang/site-www/blob/f4f49bf27ae8f3482e320b69e431c35ada6f2b6a/src/_11ty/plugins/markdown.js#L15-L16) it to start with `{:`. I had a reason at the time, but I am struggling to recall why now... If I remember, I'll document it, otherwise I might go back to the default. Thanks for asking!